default_configuration.yml

```diff
@@ -1,7 +1,6 @@
 dataSources:
   - executor: FirebaseSource
   - executor: RedditCrawler
-  - executor: YoutubeCrawler
```

Comment:
why was the YoutubeCrawler removed? Does it not work?

Reply:
We need to specify search terms, maybe there is a default one that @tonymeng1 can suggest that he used when creating it.